### PR TITLE
Added Konflux configuration for tekton-caches

### PR DESCRIPTION
**Pull request by pramodbindal**

### CODE REVIEW

File: config/konflux/tektoncd-caches.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+repository: tektoncd-caches
+components:
+- cache
```
Comment (savitaashture) [inline config/konflux/tektoncd-caches.yaml]: only one image is pushed or tekton-chache have multiple images
I see there is fetch and upload
so under components
you can add all those name of the images which are part of cache

Comment (pramodbindal) [reply config/konflux/tektoncd-caches.yaml]: that is common image for both fetch and upload

Comment (savitaashture) [reply config/konflux/tektoncd-caches.yaml]: I see okok

**Pull request merged**